pkg/network/transport/object/grpc/replication.go

```diff
@@ -0,0 +1,149 @@
+package object
```

Comment:
we have here `get`, `search`, etc files but `replication` for `Replicate`

Reply:
pretty undestandable IMO

Reply:
i meant `SEARCH` lead to `search` file, `GET` for `get`, etc. i do not say it is hard to understand but it is just not named as the corresponding rpc is

Reply:
overthinking to me but ok i'll rename the file